File: FizzBuzz.py
#For "Fizz" print numbers 1-100 for multiples of 3
#For "Buzz" print numbers 1-100 for multiples of 5
#For "FizzBuzz" print numbers 1-100 for multiples of 15

for i in range(1, 101):
    if i % 15 == 0:
        print("FizzBuzz")
        continue
    elif i % 3 == 0:
        print("Fizz")
        continue
    elif i % 5 == 0:
        print("Buzz")
        continue

    print(i)

File: FizzBuzz.js
	
	for (let i=1; i<=100; i++) //For loop
	{
		
		if (i%15 == 0) //1st if statement for multiples of 15 and printing FizzBuzz
			console.log("FizzBuzz" + " ");
		
		else if ((i%3) == 0) //2nd if statement for multiples of 3 and printing Fizz
			console.log("Fizz" + " ");			
		
		else if ((i%5) == 0) //3rd if statement for multiples of 5 and printing Buzz				
			console.log("Buzz" + " ");			
	
		else
			console.log(i + " ");			
	}